taxes_by_invoice/taxes_by_invoice.py

```diff
@@ -334,6 +332,16 @@ def prepare(cls, data):
 
             for tax in taxes:
                 records.setdefault(tax.tax, []).append(DualRecord(tax))
+
+                # If the invoice is cancelled, do not add its values to the
+                # totals
+                tax.invoice.state == 'cancelled' and tax.invoice.cancel_move
```

Comment:
This line should be removed as it does nothing

Reply:
ups!

f2f4a6b40da0d3abeaeafeaccc89f056c1295abb